DigitalOffice/Tests/UserServiceTests/Mappers/UserMapperTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using UserService.Database.Entities;
+using UserService.Mappers;
+using UserService.Mappers.Interfaces;
+using UserService.Models;
+
+namespace UserServiceUnitTests.Mappers
+{
+    public class UserMapperTests
+    {
+        private IMapper<DbUser, User> mapper;
+
+        private const string Email = "smth@emal.com";
+        private const string Message = "smth";
+        private const string FirstName = "Ivan";
+        private const string LastName = "Dudikov";
+        private const string PasswordHash = "42";
+        private const bool IsActive = true;
+        private const string Status = "Hello, world!";
+
+        private Guid userId;
+        private Guid achievementId;
+        private Guid certificateFileId;
+        private Guid pictureFileId;
+        private Guid avatarFileId;
+
+        private DbAchievement achievement;
+        private DbUserAchievement dbUserAchievement;
+        private DbUser dbUser;
+        private DbUserCertificateFile dbUserCertificateFile;
+
+        [SetUp]
+        public void SetUp()
+        {
+            mapper = new UserMapper();
+            userId = Guid.NewGuid();
+            achievementId = Guid.NewGuid();
+            certificateFileId = Guid.NewGuid();
+            pictureFileId = Guid.NewGuid();
+            avatarFileId = Guid.NewGuid();
+            achievement = new DbAchievement {Id = achievementId, Message = Message, PictureFileId = pictureFileId};
+            dbUserAchievement = new DbUserAchievement
+            {
+                Achievement = achievement, AchievementId = achievementId, User = dbUser, Time = DateTime.Now,
+                UserId = userId
+            };
+            dbUserCertificateFile = new DbUserCertificateFile
+                {CertificateId = certificateFileId, User = dbUser, UserId = userId};
+            dbUser = new DbUser
+            {
+                AchievementsIds = new List<DbUserAchievement> {dbUserAchievement}, AvatarFileId = avatarFileId,
+                CertificatesFilesIds = new List<DbUserCertificateFile> {dbUserCertificateFile}, Email = Email,
+                FirstName = FirstName, Id = userId, IsActive = IsActive, LastName = LastName, MiddleName = null,
+                PasswordHash = PasswordHash, Status = Status
+            };
+        }
+
+        [Test]
+        public void ShouldThrowArgumentNullExceptionIfArgumentOfMapIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() => mapper.Map(null));
+        }
+
+        [Test]
+        public void ShouldSuccessfulReturnUserModelAfterMapCorrectDbEntity()
```

Comment:
I think these are too complex names for tests.

Reply:
Name of the test should contain information about expected result, condition. What are the problems with complex test names?

Reply:
Can be named shorter, for example, SuccessReturnUserModelAfterMapDbEntity. I agree, this is not very important.